cmake/Dependencies.cmake

```diff
@@ -89,17 +89,16 @@ if(USE_ROCM)
     # Ask hcc to generate device code during compilation so we can use
     # host linker to link.
     foreach(gloo_rocm_arch ${GLOO_ROCM_ARCH})
-        list(APPEND HIP_CXX_FLAGS --amdgpu-target=${gloo_rocm_arch})
+      list(APPEND HIP_CXX_FLAGS --amdgpu-target=${gloo_rocm_arch})
     endforeach()
 
-    set(GLOO_HIP_INCLUDE
-      ${hip_INCLUDE_DIRS} )
+    set(HIP_HCC_FLAGS ${HIP_CXX_FLAGS})
```

Comment:
This looks like an unintentional addition here?

Reply:
no, we need this line. thx

Reply:
I see -- I only looked at the formatting commit and saw it added there, hence the question.